core/src/main/java/org/stellar/anchor/client/DefaultClientService.java

```diff
@@ -0,0 +1,86 @@
+package org.stellar.anchor.client;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.io.FilenameUtils;
+import org.stellar.anchor.api.exception.InvalidConfigException;
+import org.stellar.anchor.api.exception.SepNotFoundException;
+import org.stellar.anchor.config.ClientsConfig;
+import org.stellar.anchor.util.FileUtil;
+import org.stellar.anchor.util.GsonUtils;
+import org.yaml.snakeyaml.Yaml;
+
+public class DefaultClientService implements ClientService {
+  static final Gson gson = GsonUtils.getInstance();
+  List<CustodialClientConfig> custodialClients;
+  List<NonCustodialClientConfig> nonCustodialClients;
+
+  public static DefaultClientService fromClientsConfig(ClientsConfig clientsConfig)
+      throws InvalidConfigException {
+    return switch (clientsConfig.getType()) {
+      case FILE -> fromFile(clientsConfig.getValue());
+      case URL -> throw new InvalidConfigException("URL is not supported yet.");
+      default -> throw new InvalidConfigException(
+          String.format("clients.type:%s is not supported.", clientsConfig.getType()));
+    };
+  }
+
+  public static DefaultClientService fromFile(String filePath) throws InvalidConfigException {
+    Map<String, Object> map = parseFileToMap(filePath);
+    return createDCSFromMap(map);
+  }
+
+  private static Map<String, Object> parseFileToMap(String filePath) throws InvalidConfigException {
+    try {
+      String fileContent = FileUtil.read(Path.of(filePath));
+      return switch (FilenameUtils.getExtension(filePath).toLowerCase()) {
+        case "json" -> gson.fromJson(fileContent, Map.class);
```

Comment:
Why are we supporting JSON?

Reply:
This was based on Jamie's initial change https://github.com/stellar/java-stellar-anchor-sdk/pull/1306, and assume we want the same level of support for both clients and assets? 

If it's not necessary we can remove it.

Reply:
I personally think we should remove json in 3.0, but I don't feel strongly about it.